.github/actions/tf-setup/action.yml

```diff
@@ -50,6 +53,7 @@ runs:
         ARM_CLIENT_ID: ${{ inputs.azure-client-id }}
         ARM_TENANT_ID: ${{ inputs.azure-tenant-id }}
         ARM_SUBSCRIPTION_ID: ${{ inputs.azure-subscription-id }}
+        ARM_OBJECT_ID: ${{ inputs.azure-object-id }}
```

Comment:
we will still need to to add this input as a `.tfvar` from the "Load Input Variables" step, under the `run`. This is if we are going the route and not using `getenv()`

Reply:
oh wait.. does the the `azurerm_postgresql_flexible_server` resource not need the `object_id`?

Reply:
it looks like the `tenant_id` and the `client_id` are needed though right?

Reply:
I will use it for the upcoming PR for vault.

Reply:
These have been removed.  May be added with the next PR with masking the database credentials.